blockchain/source/migrations/2_deploy_v1.js

```diff
@@ -134,21 +135,25 @@ async function deploySidechain (deployer, network, accounts) {
     // write
     await ahm.write('sidechainSNMAddress', SNM.address, { gasPrice: 0 });
 
-    await deployer.deploy(Multisig, MSOwners(network, accounts), MSRequired(network), { gasPrice: 0 });
-    let multiSig = await Multisig.deployed();
-
     await deployer.deploy(OracleMultisig, MSOwners(network, accounts), MSRequired(network), { gasPrice: 0 });
     let oracleMS = await OracleMultisig.deployed();
 
+    await deployer.deploy(GateKeeperMultisig, MSOwners(network, accounts), MSRequired(network), { gasPrice: 0 });
+    let gkMS = await GateKeeperMultisig.deployed();
+
+    await deployer.deploy(Multisig, MSOwners(network, accounts), MSRequired(network), { gasPrice: 0 });
+    let multiSig = await Multisig.deployed();
+
     await ahm.write('blacklistAddress', bl.address, { gasPrice: 0 });
     await ahm.write('marketAddress', market.address, { gasPrice: 0 });
     await ahm.write('profileRegistryAddress', pr.address, { gasPrice: 0 });
     await ahm.write('oracleUsdAddress', oracle.address, { gasPrice: 0 });
     await ahm.write('gatekeeperSidechainAddress', gk.address, { gasPrice: 0 });
     await ahm.write('multiSigAddress', multiSig.address, { gasPrice: 0 });
     // compatibility
-    await ahm.write('migrationMultSigAddress', multiSig.address, { gasPrice: 0 });
+    await ahm.write('migrationMultiSigAddress', multiSig.address, { gasPrice: 0 });
```

Comment:
you're write the same address for `migrationMultiSigAddress` and `multiSigAddress` ?

Reply:
yep, that's right

Reply:
ok, maybe it is not a good idea, if we still will have several multisigs in future

Reply:
ok, it's really right.